packages/react-i18n/src/index.ts

```diff
@@ -0,0 +1,7 @@
+export {default as Provider, Props as ProviderProps} from './Provider';
+export {default as Manager} from './manager';
```

Comment:
why is `Provider.tsx` capitalized, but `manager.ts` is lowercase? Is it just because one is react and one isn't? Do we document that convention anywhere?

Reply:
Yes that is why they are cased the way they are. Opened an issue to document file naming conventions: https://github.com/Shopify/web-foundation/issues/59